packages/agreement-process/src/services/readmodel/readModelService.ts

```diff
@@ -56,9 +56,7 @@ const makeFilter = (
       return undefined;
     });
 
-const getAgreementsFilters = (
-  filters: AgreementQueryFilters
-): { $match: object } => {
+const getAgreementsFilters = (filters: AgreementQueryFilters): Document => {
```

Comment:
I have doubt about it, the  type Document in `mongodb` is a generic shape for all possible objects in a Mongo collection. It is defined as: 
```
export declare interface Document {
    [key: string]: any;
}
```
obviously it match returned object shape, but reading this signature could be ambiguous to understand that returned `Document` is a mongo filter with `$match` operator to use for aggregation pipeline. 
For me we have two choices: investigate if there is an ad hoc type for this filter or declare our custom type for it. What do you think about it ?

Reply:
I think we can also revert this change, since we only use it here at the moment we can leave it inline.
I substituted it with `Document` without thinking too much, but I notice now that it was a stricter type :)